src/ray/util/util.h

```diff
@@ -382,4 +382,16 @@ void QuickExit();
 /// \return the foramtted value
 std::string FormatFloat(float value, int32_t precision);
 
+// Insert default labels to node labels.
+absl::flat_hash_map<std::string, std::string> GetFinalNodeLabels(
+    const absl::flat_hash_map<std::string, std::string> &labels,
+    const std::string &node_id,
+    const std::string &node_ip);
+
+#define RAY_LABEL_KEY_PREFIX "ray.io/"
```

Comment:
Instead of macro, can we define it as a const string

Reply:
Can't use a const string inststead of marco,  will complile error.